src/main/java/com/mercadopago/client/payment/PaymentCaptureRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.mercadopago.client.payment;
+
+import java.math.BigDecimal;
+import lombok.Builder;
+
+/** PaymentCaptureRequest class. */
+@Builder
+public class PaymentCaptureRequest {
```

Comment:
Acho que não tem necessidade de usar builder em classes com poucos atributos. O que acha?
Além disso, não faltou `@Data` aqui?

Reply:
Essa request foi criar apenas para passar os dados já settados pro payload quando for realizar uma requisição de cancelamento do pagamento, ou seja, não haverá nenhuma manipulação dos dados, por este motivo não vi a necessidade de colocar o @Data.